### ISSUE TITLE
add `sbt new` command to readme

### ISSUE THREAD

**Issue by patriknw** [akka/akka-http-scala-seed.g8#2]
I think it's convenient to be able to copy the full `sbt new` command from the readme.

```
sbt -Dsbt.version=0.13.15 new https://github.com/akka/akka-http-scala-seed.g8
```

Maybe the readme should have even more instructions, e.g. for inspiration: https://www.lagomframework.com/get-started-java-sbt.html

@jlprat can you fix this while you are touching things in these repos?

**Comment by jlprat** [comment]
Yes I can extend the README with:
- Sbt command to create a new project
- Explanation of the different parameters one can provide
- What it is inside (in this case 2 different servers one using HttpApp, the other doing it manually)
- How to run it

**Comment by ktoso** [comment]
Those points sound good, thanks @jlprat!
An one liner "copy paste this to get started" and explanations as you mentioned would be very good.

**Comment by jlprat** [comment]
Will do :)

**Comment by jlprat** [comment]
Solved via #3
